.mci.yml

```diff
@@ -79,7 +79,7 @@ functions:
                 cd mongo-c-driver
                 git checkout 1.3.4
                 rm -rf /data/tmp/c-driver-install
-                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align ${cdriver_configure_flags}
+                ./autogen.sh --prefix="/data/tmp/c-driver-install" --enable-tests=no --enable-examples=no --with-libbson=bundled --disable-extra-align --enable-debug --enable-optimizations --disable-shm-counters --disable-static --disable-dependency-tracking --with-pic --disable-automatic-init-and-cleanup ${cdriver_configure_flags}
```

Comment:
The disable-automatic-init-and-cleanup turns off mongoc's use of gcc's constructor/destructor attributes, so that cleanup happens only when *we* call it, not at some indeterminate time (possibly before we call it, leading to a runtime exception when we call it again).